src/components/admin/HistoryViewer/HistoryList.tsx

```
@@ -0,0 +1,240 @@
+
+    const handleCopyHash = (e: React.MouseEvent) => {
+        e.stopPropagation();
+        navigator.clipboard.writeText(commit.sha);
+        setCopied(true);
+        setTimeout(() => setCopied(false), 2000);
+    };
+
+    return (
+        <li className="border-b last:border-0 border-sidebar-border/50">
+            <div
+                role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)